docs/select/README.md

```diff
@@ -0,0 +1,86 @@
+# S3 Select API Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+In Minio, we can store an object as large as 5 TiB. Traditional retrieval of objects is always as whole entities, i.e if you ask for a 5 GiB object, you will always get 5 GiB. [S3 Select API](https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectSELECTContent.html) allows us to retrieve a subset of data by using simple SQL expressions. By using Select to retrieve only the data needed by the application, drastic performance improvements can be achieved.
+
+## 1. Prerequisites
+- Install Minio Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
+- Familiarity with [AWS S3 API](https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html).
+- Familiarity with Python and installing dependencies.
+
+## 2. Install boto3
+Install `aws-sdk-python` from AWS SDK for Python official docs [here](https://aws.amazon.com/sdk-for-python/)
```

Comment:
Can this be also done using Minio Python Client SDK?

Reply:
In future @ugurtigli

Reply:
ok. Also, should we call it "Minio SELECT Object API" or something similar to show this is our implementation?